src/main/java/com/minecolonies/coremod/entity/ai/mobs/EntityChiefBarbarian.java

```diff
@@ -0,0 +1,189 @@
+    private static final int              TIME_TO_COUNTDOWN           = 30;
+    private static final int              COUNTDOWN_SECOND_MULTIPLIER = 4;
+    private              int              currentCount                = 0;
+
+    /**
+     * defines the default values for the Entity's attributes.
+     */
+    private static final double FOLLOW_RANGE          = 35.0D;
+    private static final double MOVEMENT_SPEED        = 0.2D;
+    private static final double ATTACK_DAMAGE         = 2.0D;
+    private static final double ARMOR                 = 2.0D;
+    private static final double BARBARIAN_BASE_HEALTH = 25;
+
+    /**
+     * Defines the default values for the various AI Task's priorities.
+     */
+    private static final int   PRIORITY_ZERO      = 1;
+    private static final int   PRIORITY_TWO       = 2;
+    private static final int   PRIORITY_THREE     = 3;
+    private static final int   PRIORITY_FOUR      = 4;
+    private static final int   PRIORITY_FIVE      = 5;
+    private static final int   PRIORITY_SIX       = 6;
+    private static final int   PRIORITY_EIGHT     = 8;
+    private static final float MAX_WATCH_DISTANCE = 8.0F;
+
+    /**
+     * Values used to choose whether or not to play sound
+     */
+    private static final int OUT_OF_ONE_HUNDRED = 100;
+    private static final int ONE                = 1;
+
+    /**
+     * Constructor method for entity
+     *
+     * @param worldIn The world that the entity is in
+     */
+    public EntityChiefBarbarian(final World worldIn)
+    {
+        super(worldIn);
+        this.getAlwaysRenderNameTag();
+    }
+
+    @Override
+    protected void applyEntityAttributes()
+    {
+        super.applyEntityAttributes();
+        this.getEntityAttribute(SharedMonsterAttributes.FOLLOW_RANGE).setBaseValue(FOLLOW_RANGE);
+        this.getEntityAttribute(SharedMonsterAttributes.MOVEMENT_SPEED).setBaseValue(MOVEMENT_SPEED);
+        this.getEntityAttribute(SharedMonsterAttributes.ATTACK_DAMAGE).setBaseValue(ATTACK_DAMAGE);
+        this.getEntityAttribute(SharedMonsterAttributes.ARMOR).setBaseValue(ARMOR);
+        this.getEntityAttribute(SharedMonsterAttributes.MAX_HEALTH).setBaseValue(this.getHealthBasedOnRaidLevel());
+    }
+
+    @Nullable
+    @Override
+    public IEntityLivingData onInitialSpawn(final DifficultyInstance difficulty, @Nullable final IEntityLivingData livingdata)
+    {
+        this.setItemStackToSlot(EntityEquipmentSlot.MAINHAND, new ItemStack(ModItems.chiefSword));
+        return super.onInitialSpawn(difficulty, livingdata);
+    }
+
+    /**
+     * Done so that the Chief does not drop his equipped Chief Sword, because it's supposed to be rare
+     *
+     * @param wasRecentlyHit  Boolean value, of whether it was recently hit
+     * @param lootingModifier The modifier applied by a looting enchantment
+     */
+    @Override
+    protected void dropEquipment(final boolean wasRecentlyHit, final int lootingModifier)
+    {
```

Comment:
"""
/**
     * Done so that the Chief does not drop his equipped Chief Sword, because it's supposed to be rare
     *
     * @param wasRecentlyHit  Boolean value, of whether it was recently hit
     * @param lootingModifier The modifier applied by a looting enchantment
     */
    @Override
    protected void dropEquipment(final boolean wasRecentlyHit, final int lootingModifier)
    {
    }
"""